DashKitUI/Views/DashSettingsView.swift

```diff
@@ -19,11 +19,18 @@ struct DashSettingsView: View  {
    @State private var showingDeleteConfirmation = false
 
    @State private var showSuspendOptions = false;
+   
+   @State private var showSyncTimeOptions = false;
 
    @Environment(\.guidanceColors) var guidanceColors
    @Environment(\.insulinTintColor) var insulinTintColor
-
+   
    weak var navigator: DashUINavigator?
+   
+   var pumpTimeIsOffset: Bool {
```

Comment:
[nit] Hm...would've expected this policy check to be in the `viewModel` (and under unit test...)